CHANGELOG.md

```diff
@@ -24,6 +24,9 @@
       * XAML IntelliseSense completions for name properties like Setter.TargetName
 
 # 2.54.x
+* Update Razor to 9.0.0-preview.24531.4 (PR: [#7723](https://github.com/dotnet/vscode-csharp/pull/7723))
```

Comment:
I think this is in the wrong section, we're on 2.55

Reply:
woops , thanks